tools/commands/analyze_plugins.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+import os
+import commands.command_utils as command_utils
+
+
+def parse_args(args):
+    parser = command_utils.get_options_parser(
+        plugins=True, exclude=True, run_on_changed_packages=True, base_sha=True, command='analyze')
+    return parser.parse_args(args)
+
+
+def _run_analyze_plugins(plugin):
```

Comment:
How about:
```suggestion
def _run_analyze_plugins(plugin_dir):
```


`plugin` is generally used for the name of the plugin, for example line 31.
I would appreciate if you also change `_build_examples(plugin)` to `_build_examples(plugin_dir)` in `_build_example.py`.

Reply:
please, check again, I applied it to all the places you mentioned.